pmem CI failing due docker limit

The docker limit issue hit PMEM CI with
```
22:07:38 Creating rootfs for ubuntu
22:07:38 /home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/rootfs.sh -o 2.2.0-alpha1-e155fb2b199c6ff60eed028e6c04dbe47d544823 -r /home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/ubuntu_rootfs ubuntu
22:07:38 INFO: rootfs_lib.sh file found. Loading content
22:07:38 INFO: Detecting agent rust version
22:07:38 INFO: Get rust version from /home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/../../../versions.yaml
22:07:39 Required rust version: 1.47.0
22:07:39 INFO: Detecting musl version
22:07:39 INFO: Get musl version from /home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/../../../versions.yaml
22:07:39 Required musl version: 1.1.23
22:07:39 ~/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/rootfs-builder/ubuntu ~/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder
22:07:39 ~/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder
22:07:39 Sending build context to Docker daemon  10.75kB

22:07:39 Step 1/6 : ARG IMAGE_REGISTRY=docker.io
22:07:39 Step 2/6 : FROM ${IMAGE_REGISTRY}/ubuntu:20.04
22:07:40 20.04: Pulling from library/ubuntu
22:07:40 toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
22:07:40 Makefile:93: recipe for target '/home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/.ubuntu_rootfs.done' failed
22:07:40 make: *** [/home/jenkins/workspace/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/.ubuntu_rootfs.done] Error 1
22:07:40 [install_kata_image.sh:50] ERROR: sudo -E USE_DOCKER=1 DISTRO=ubuntu make -e image
22:07:40 [install_kata.sh:22] ERROR: .ci/install_kata_image.sh 
22:07:40 Build step 'Execute shell' marked build as failure
```
/cc @devimc

--- a/integration/kubernetes/init.sh
+++ b/integration/kubernetes/init.sh
@@ -71,7 +71,7 @@ esac
 check_processes
 
 echo "Build custom stress image"
-registry_image="docker.io/library/registry"
+registry_image="docker.io/library/registry:2"
 arch=$("${SCRIPT_PATH}/../../.ci/kata-arch.sh")
 if [[ "${arch}" == "ppc64le" || "${arch}" == "s390x" ]]; then
 	# that image is not built for these architectures

--- a/integration/pmem/pmem_test.sh
+++ b/integration/pmem/pmem_test.sh
@@ -40,7 +40,6 @@ fi
 
 init() {
 	${dir_path}/../../integration/kubernetes/init.sh
-	docker run -d -p 5000:5000 --restart=always --name registry registry:2
 }
 
 cleanup() {
@@ -59,10 +58,6 @@ cleanup() {
 	rm -rf pmem-csi
 
 	${dir_path}/../../integration/kubernetes/cleanup_env.sh
-
-	docker rm -f $(docker ps -aq)
-	rid=$(docker images -q registry:2)
-	docker rmi -f $(docker images -q | sed '/'$rid'/d')
 }
 
 run_test() {

